apps/lib/blockly/hy_am.js

```diff
@@ -249,7 +249,7 @@ Blockly.Msg.MATH_RANDOM_FLOAT_HELPURL = "http://en.wikipedia.org/wiki/Random_num
 Blockly.Msg.MATH_RANDOM_FLOAT_TITLE_RANDOM = "պատահական մասն";
 Blockly.Msg.MATH_RANDOM_FLOAT_TOOLTIP = "Վերադարձնել պատահական խմբակցության միջեւ 0.0 (ներառյալ) եւ 1.0 (բացառիկ):";
 Blockly.Msg.MATH_RANDOM_INT_HELPURL = "http://en.wikipedia.org/wiki/Random_number_generation";
-Blockly.Msg.MATH_RANDOM_INT_TITLE = "Պատահական ամբողջ թիվ է% 1% 2";
+Blockly.Msg.MATH_RANDOM_INT_TITLE = "Պատահական ամբողջ թիվ է% %1 %2";
```

Comment:
Actually, is the `%` before the ` %1` possibly a stray one that should be removed?